proto/minder/v1/minder.proto

```diff
@@ -2290,33 +2332,75 @@ message Profile {
         // id is optional and use for updates to match upserts as well as read operations. It is ignored for creates.
         string id = 1;
         // entity is the entity to select.
-        string entity = 2;
+        string entity = 2 [
+            (buf.validate.field).string = {
+                pattern: "^[a-z]+(_[a-z]+)*$",
+                min_len: 1,
+                max_len: 200,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
         // expr is the expression to select the entity.
-        string selector = 4;
+        string selector = 4 [
+            (buf.validate.field).string = {
+                max_len: 200,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
         reserved 5;
         reserved "comment";
         // description is the human-readable description of the selector.
-        string description = 6;
+        string description = 6 [
+            (buf.validate.field).string = {
+                pattern: "^[A-Za-z][-/.!?,:;[:word:] ]*$",
+                max_len: 1000,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
     }
 
     repeated Selector selection = 14;
 
     // whether and how to remediate (on,off,dry_run)
     // this is optional and defaults to "off"
-    optional string remediate = 8;
+    optional string remediate = 8 [
+        (buf.validate.field).string = {
+            in: ["on", "off", "dry_run"]
+        },
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+    ];
 
     // whether and how to alert (on,off,dry_run)
     // this is optional and defaults to "on"
-    optional string alert = 9;
+    optional string alert = 9 [
+        (buf.validate.field).string = {
+            in: ["on", "off", "dry_run"]
+        },
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+    ];
 
     // type is a placeholder for the object type. It should always be set to "profile".
-    string type = 10;
+    string type = 10 [
+        (buf.validate.field).string = {
+            pattern: "profile",
+        }
+    ];
 
     // version is the version of the profile type. In this case, it is "v1"
-    string version = 11;
+    string version = 11 [
+        (buf.validate.field).string = {
+            pattern: "^v\\d$",
+        }
```

Comment:
This is never empty, correct?

Reply:
Yeah, I checked all of the profiles I could find and it's set everywhere 👍